libs/coin-modules/coin-evm/src/__tests__/coin-tester/scenarios/ethereum.ts

```diff
@@ -0,0 +1,200 @@
+import Eth from "@ledgerhq/hw-app-eth";
+import { BigNumber } from "bignumber.js";
+import { ethers, providers } from "ethers";
+import { killSpeculos, spawnSigner } from "@ledgerhq/coin-tester/docker";
+import { encodeTokenAccountId } from "@ledgerhq/coin-framework/account/index";
+import { Scenario, ScenarioTransaction } from "@ledgerhq/coin-tester/main";
+import { getCryptoCurrencyById } from "@ledgerhq/cryptoassets/currencies";
+import { buildAccountBridge, buildCurrencyBridge } from "../../../bridge/js";
+import { makeAccount } from "../../fixtures/common.fixtures";
+import { EvmNftTransaction, Transaction as EvmTransaction } from "../../../types";
+import resolver from "../../../hw-getAddress";
+import { setCoinConfig } from "../../../config";
+import {
+  ethereum,
+  ERC20Interface,
+  USDC_ON_ETHEREUM,
+  ERC721Interface,
+  ERC1155Interface,
+} from "../helpers";
+import { clearExplorerAppendix, getLogs, setBlock } from "../indexer";
+import { killAnvil, spawnAnvil } from "../docker";
+
+const scenarioSendTransaction: ScenarioTransaction<Partial<EvmTransaction>> = {
+  name: "Send ethereum",
+  amount: new BigNumber(100),
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+};
```

Comment:
We forgot to add expect to each transaction in order to verify the result of the sync after each broadcast.